src/screens/Gurbani/Line.tsx

```diff
@@ -0,0 +1,69 @@
+import { toUnicode } from 'gurmukhi-utils'
+import React from 'react'
+import { StyleSheet, View } from 'react-native'
```

Comment:
```suggestion
import React from 'react'
import { StyleSheet, View } from 'react-native'
import { toUnicode } from 'gurmukhi-utils'
```
I think there should be an eslint rule that force react import as first thing in file but fwiw it is gonna get removed once new RN with react 17 is out.

Reply:
Yeah i agree, not sure what rule there could be though. Perhaps we could also just turn on alaphabetical ESLint import ordering and just get used to it too? Thoughts? @bhajneet

Reply:
We are using [`simple-import-sort`](https://github.com/lydell/eslint-plugin-simple-import-sort#example) plugin but right now we just have it grouped into sections [see docs](https://github.com/lydell/eslint-plugin-simple-import-sort#imports). Don't know if it does already but it would be great if it can alphabetically sort grouped sections.